src/transformers/models/llama/configuration_llama.py

```diff
@@ -151,6 +151,11 @@ class LlamaConfig(PretrainedConfig):
         "layers.*.mlp.up_proj": "colwise",
         "layers.*.mlp.down_proj": "rowwise",
     }
+    base_model_pp_plan = {
+        "input_processing": {"embed_tokens": "inputs_embeds"},
+        "decoder_stack": {"layers": "layer_outputs"},
+        "output_processing": {"norm": "hidden_states"},
```

Comment:
I was thinking more of an ordereddict or a list here.
Because unlike TP where order doesnt matter much, we need to know the order of operations happening after the last decoder layer, and let the other libraries decide in how they want to optimize that

Reply:
Since `dict`s have been ordered since Python 3.7, it's probably safe to assume that all `dict`s are ordered